docs/notebooks/structure_function_showcase.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    # ens.from_source_dict(source_dict=data_dict, column_mapper=manual_colmap)
</pre><p>As we discussed directly, this fails because 2d arrays are attempting to be passed along</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Fixed this.